form/form_dependencies.rst

```diff
@@ -71,36 +71,41 @@ Alternatively, you can define your form class as a service. This is a good idea
 you want to re-use the form in several places - registering it as a service makes
 this easier.
 
-Suppose you need to access the ``doctrine.orm.entity_manager`` service so that you
+Suppose you need to access the ``EntityManager`` service so that you
```

Comment:
There is also an EntityManagerInterface result:

```
Doctrine\ORM\EntityManagerInterface             alias for "doctrine.orm.default_entity_manager"
```

But you're right. Saying:

> I need to access the EntityManagerInterface service

sounds weirder. Technically the concrete instance *is* `EntityManager`, but it's true that you don't see that unless you specifically look at the details for `doctrine.orm.default_entity_manager`. 

So there's no "EntityManager service", but there *is* an EntityManager object in the container... which is what the user wants (and this is a service). 

Anyways, I've made a change, let me know what you think - we should continue tweaking. I think it's important because we need to be able to find clear language to explain things, otherwise there's a problem :)